packages/@markuplint/ml-core/src/ml-dom/helper/walkers.ts

```diff
@@ -1,10 +1,6 @@
-import { AnonymousNode } from '../types';
+import { AnonymousNode, SyncWalker } from '../types';
 import { RuleConfigValue } from '@markuplint/ml-config';
 
-export type Walker<T extends RuleConfigValue, O = null, N = AnonymousNode<T, O>> = (node: N) => Promise<void>;
```

Comment:
I moved them into types files to be constantly and exports them all to reuse them in other packages.